Lib/_osx_support.py

```diff
@@ -481,6 +511,8 @@ def get_platform_osx(_config_vars, osname, release, machine):
 
             if len(archs) == 1:
                 machine = archs[0]
+            elif archs == ('arm64', 'x86_64'):
+                machine = 'universal2'
```

Comment:
@ronaldoussoren I'm uncertain whether I should leave in the changes to allow `universal2`? I don't believe Homebrew requires this, nor would we (Dropbox).

I also don't think `universal2` would be very practical given I am not back porting the weak linking changes.